mmcv/transforms/utils.py

```diff
@@ -80,7 +81,7 @@ def __call__(self, *args, **kwargs):
 
     def __get__(self, obj, cls):
         self.instance_ref = weakref.ref(obj)
-        return self
+        return copy.copy(self)
```

Comment:
```suggestion
        # Return a copy to avoid multiple transform instances sharing
        # one `cache_randomness` instance, which may cause data races
        # in multithreading cases.
        return copy.copy(self)
```